File: LAB6/PYTHON/lab6_1.py

n = int(input("Enter value of n: "))
x = int(input("Enter value of x: "))

factorial = lambda num: 1 if num == 0 else num * factorial(num-1)

list = [n/(factorial(i)) for i in range(x+1)]
result = 1 + sum(list)

print("The result is : " , result)

File: LAB6/C++/lab6_1.cpp
#include <iostream>
#include <cmath>
using namespace std;


int factorial(int a) {
    return (a == 0) ? 1 : a * factorial(a - 1);
}

int main () {
    int n;
    int x;

    cout<<"Enter n: ";
    cin>>n;
    cout<<"Enter x: ";
    cin>>x;

    double * b = new double[x];

    for (int i = 1; i <= x; i++) {
        b[i-1] = pow(n, i) / factorial(i);
    }

    double result = 1;

    for (int i = 0; i < x; i++) {
        result += b[i];
    }

    cout<<"The result is : " << result << endl;
    return 0;
}